helix-rest/pom.xml

```diff
@@ -148,6 +148,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard.metrics</groupId>
+      <artifactId>metrics-jersey2</artifactId>
```

Comment:
Where is the new ivy def for metrics-jersey2? Do we need that too?

Reply:
I thought about it. I don't see `jersey-server` is added to ivy file. Instead, we add the dependency to our MP wrapper. So I thought we could also add `metrics-jersey2` to the MP directly. Otherwise, we would need to add all the dependencies in pom to ivy.

Reply:
As long as you are ensured that anyone without accessing "MP" and using ivy can see the same feature, then it should be fine.
But from what you described, I feel it is a potential problem.

Reply:
As long as users are not using ivy, it is fine because we've defined all the required dependencies in pom. If they are using ivy without adding dependencies, it's been a problem because dependencies like `jersey-server` are not even in rest ivy.